src/main/scala/org/constellation/primitives/Genesis.scala

```diff
@@ -2,81 +2,66 @@ package org.constellation.primitives
 
 import java.security.KeyPair
 
-import better.files.File
-import cats.effect.{ContextShift, IO}
+import cats.effect.{IO, _}
 import cats.implicits._
 import com.typesafe.scalalogging.StrictLogging
-import constellation._
+import io.chrisdavenport.log4cats.SelfAwareStructuredLogger
 import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
-import io.chrisdavenport.log4cats.{Logger, SelfAwareStructuredLogger}
-import org.constellation.domain.consensus.ConsensusStatus
+import org.constellation.DAO
 import org.constellation.keytool.KeyUtils
 import org.constellation.primitives.Schema._
 import org.constellation.schema.Id
-import org.constellation.serializer.KryoSerializer
 import org.constellation.util.AccountBalance
-import org.constellation.{ConfigUtil, ConstellationExecutionContext, DAO}
-
-import scala.util.{Failure, Success, Try}
 
 object Genesis extends StrictLogging {
 
   implicit val unsafeLogger: SelfAwareStructuredLogger[IO] = Slf4jLogger.getLogger[IO]
 
-  final val CoinBaseHash = "coinbase"
-  final val DebtKey = KeyUtils.makeKeyPair()
+  final val Coinbase = "coinbase"
+  // TODO: That key should be provided externally by the genesis creator.
+  // We can use Node's KeyPair as well.
+  final val CoinbaseKey = KeyUtils.makeKeyPair()
```

Comment:
@buckysballs that is the interesting part for you.
It is the key used for creating initial transactions and genesis block along with it's child blocks. For now it is random but as we talked yesterday it should be either provided externally or maybe we should use there Node's keypair 🤔